core/lib/store-assets.ts

```diff
@@ -26,7 +26,7 @@ const cdnImageUrlBuilder = (sizeSegment: string, source: string, path: string):
  * @returns {string} The full URL to the content asset.
  */
 export const contentAssetUrl = (path: string): string => {
-  return `https://${buildConfig.get('urls').cdnUrl}/s-${storeHash}/content/${path}`;
+  return `https://${buildConfig.get('urls').cdnUrls[0]}/s-${storeHash}/content/${path}`;
```

Comment:
As mentioned in previous comment, we should handle the empty case

Reply:
Updated to use `.at`